README.md

```diff
@@ -203,6 +215,25 @@ To evaluate a finetuned model, use the following command from the project basedi
 - [Pretrained ImageNet weights](https://github.com/amirbar/DETReg/releases/download/1.0.0/checkpoint_imagenet.pth) 
 - [Finetuned COCO weights](https://github.com/amirbar/DETReg/releases/download/1.0.0/full_coco_finetune.pth)
 
+### Using hubconfig for evaluation:
+1. Pretrained model with MDef-DETR proposals:
+```bash
+checkpoints_path = "./saved_models/DETReg_top30_in100_mdef_detr.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=91,
+                                 object_embedding_loss=True, checkpoints_path=checkpoints_path)
+```
+2. Fine tune on 10% of Pascal:
+```bash
+checkpoints_path = "./saved_models/DETReg_fine_tune_10pct_pascal_mdef_detr.pth"
```

Comment:
Let's make sure that the training configs that we are providing save the models with the same names as mentioned here